notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -194,7 +194,7 @@
         "    \n",
```

Comment:
<p>drop Immutable</p><p> </p><p>Else do not set them. -&gt;</p><p>Otherwise do not set them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>done</p>